tests/test_interface.py

```diff
@@ -119,6 +119,13 @@ def set_mtu(self, interface, mtu):
         tbl.set(interface, fvs)
         time.sleep(1)
 
+    def set_dhcp_rate_limit(self, interface, dhcp_rate_limit):
+        tbl_name = "PORT"
+        tbl = swsscommon.Table(self.cdb, tbl_name)
+        fvs = swsscommon.FieldValuePairs([("dhcp_rate_limit", dhcp_rate_limit)])
+        tbl.set(interface, fvs)
+        time.sleep(20)
```

Comment:
Please fine-tune this large sleep intervals. These can cause vstest execution time to increase over time.